src/PowerShellEditorServices/Services/PowerShellContext/PowerShellContextService.cs

```diff
@@ -20,6 +20,7 @@
 using Microsoft.PowerShell.EditorServices.Logging;
 using Microsoft.PowerShell.EditorServices.Services.PowerShellContext;
 using Microsoft.PowerShell.EditorServices.Utility;
+using SMA = System.Management.Automation;
```

Comment:
For the other places where we're using an internal using statement to fix this namespace type collision, let's change them to this form of `SMA.PowerShell` to be consistent about the clear typing.